### PR TITLE
get_dist 

### PR DESCRIPTION
**Pull request by Delkhaz**
added get_dist skeleton w comments. 
removed std::collection::hashmap - not needed anymore

### CODE REVIEW

File: examples/firefly.rs [hunk 1]
```diff
@@ -70,6 +67,14 @@ impl FireflyWorld {
         self.firefly_swarm.push(firefly);
     }
 
+    fn getDist(fireFlyA:&Firefly, fireFlyB: &Firefly) -> f32 {
```
Comment (starfys) [inline examples/firefly.rs]: The naming here should be changed to better conform to rust's style. Try changing this line to 
```rust
fn get_dist(firefly_a: &Firefly, firefly_b: &Firefly) -> f32
```

Comment (starfys) [reply examples/firefly.rs]: Actually, a more ergonomic way would be to make this a function of firefly, but that's a matter of opinion. Something like
```rust
impl Firefly {
    fn distance_to(&self, &other: Firefly) -> f32 {
        ...
    }
}
```
But I'm not 100% sure. @sremedios  opinions?

Comment (Delkhaz) [reply examples/firefly.rs]: I will fix naming to better conform to rust style :rust: ! 
regarding making it a function of firefly - @sremedios ?

**Comment by sremedios** [comment]
I thought that the factor of how far a signal should go is a world thing instead of an entity thing. This distance calculation will be used to determine who the world will call receive_message() on.

**Comment by Delkhaz** [comment]
get_dist function is complete:
calculates Euclidean distance between two fireflys in n dimensional space

test_get_dist function is also complete and passes.

**Review by sremedios — APPROVED** [review]

**Pull request merged**